tasks/main.yml

```diff
@@ -14,7 +14,34 @@
     # it.
     state: latest  # noqa 403
     update_cache: yes
-  when: ansible_os_family == "Debian"
+  when: >
+    ansible_os_family == "Debian" and
+    ansible_distribution_release != "kali-rolling"
+
+- name: Reinstall cryptsetup-initramfs so upgrade doesn't fail
+  apt:
+    name: 'cryptsetup-initramfs'
+    state: latest  # noqa 403
+    update_cache: yes
+  when: ansible_distribution_release == "kali-rolling"
+
+- name: apt-get update
+  shell: apt-get update  # noqa 303 305
+  args:
+    warn: false
+  when: ansible_distribution_release == "kali-rolling"
+  tags: molecule-idempotence-notest
+
+# Upgrading with shell command instead of apt ansible module
+# See https://github.com/ansible/ansible/issues/63134 for more info
+- name: Upgrade with noninteractive frontend to handle GUI prompts
+  shell: apt-get -yy upgrade  # noqa 303 305
+  when: ansible_distribution_release == "kali-rolling"
+  args:
+    warn: false
+  environment:
+    DEBIAN_FRONTEND: noninteractive
+  tags: molecule-idempotence-notest
```

Comment:
See previous comment regarding `molecule-idempotence-notest`.

Reply:
Addressed in commit 9f51229